lib/user.rb

```diff
@@ -0,0 +1,26 @@
+require 'pg'
+require_relative 'database_connection'
+
+class User
+
+  attr_reader :id, :name, :email, :username, :password
```

Comment:
Need to ensure any password stored is encrypted, otherwise it is very easily accessible through this attr_reader